### PR TITLE
added landing page

### PR DESCRIPTION
**Pull request by gauravyadav-wd**

### CODE REVIEW

File: booking-page/index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+          <h3>IMOX, Noida</h3>
+        </div>
+        <div class="info">
+          <span class="material-symbols-outlined"> info </span>
+          <p>INFO</p>
+        </div>
+        <div class="timings">
+          <div class="timing">
+            <p>8:15</p>
+          </div>
```
Comment (baibhav-celestial) [inline booking-page/index.html:87]: Make the timings dynamic

File: booking-page/index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+    <div class="modal hidden">
+      <h3 class="modal-heading">How many Seats?</h3>
+      <!-- <img src="../img/hp.jpg" width="100" alt="image" /> -->
+      <div class="seat-nums">
+        <div class="seat">1</div>
+        <div class="seat active-seat">2</div>
+        <div class="seat">3</div>
+        <div class="seat">4</div>
+        <div class="seat">5</div>
```
Comment (baibhav-celestial) [inline booking-page/index.html:118]: These seat numbers and prices should be dynamic

File: booking-page/index.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+FRI <br />
+<span class="day"> 13 </span><br />
+OCT
+</div>`;
+
+// Rendering Dates from today to next 7 days (today is highlighted by default)
+
+dates.innerHTML = "";
+
+for (let i = 0; i < 7; i++) {
```
Comment (baibhav-celestial) [inline booking-page/index.js:40]: avoid using variable names like i. use proper naming conventions and try to use modern looping methods.

File: booking-page/index.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+// Rendering Timings for current day (not rendering elapsed time)
+
+// const timeHtml = `<div class="timing">
+// <p>8:15</p>
+// </div>`;
+// const movieTimings = ["8:15", "11:30", "14:40", "15:30", "20:00", "23:10"];
+// const activeDate = document.querySelector(".active-date");
+// const timings = document.querySelector(".timings");
+// console.log(timings);
+
```
Comment (baibhav-celestial) [inline booking-page/index.js]: If commented code is not needed please remove it.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+            <a href="" class="view-all-btn"
+              ><span class="material-symbols-outlined"> arrow_right_alt </span>
+            </a>
+          </div>
+          <div class="movies">
+            <div class="movie">
+              <img src="./img/hp.jpg" width="200" alt="" />
+              <h3>Harry Potter</h3>
+            </div>
+            <div class="movie">
```
Comment (baibhav-celestial) [inline index.html:75]: Use array or any object to print these divs. make them dynamic.

File: index.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+//   const options = {
+//     method: "GET",
+//     headers: {
+//       accept: "application/json",
+//       Authorization:
+//         "Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIzZmRkZDBhNmNjMzBhYWU2OWI4MzE0YTcyNWExNGQ0YSIsInN1YiI6IjY1MjdjNjA5MWYzZTYwMDExYzQ4ZTUxMSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.tpZsJkzTcCg9ZHaVuKmQFXNQ6vLPjoPkwfM_ggJoaVg",
+//     },
+//   };
+//   fetch(
+//     "https://api.themoviedb.org/3/movie/popular?language=en-US&page=1",
```
Comment (baibhav-celestial) [inline index.js]: This file has no code, please explain the need of this file.

Comment (gauravyadav-wd) [reply index.js]: this was for api testing purpose. I am removing it

File: index.html (hunk 5 above)
Comment (gauravyadav-wd) [reply index.html:75]: mayank is working on it. he is handling API part.

File: booking-page/index.js (hunk 4 above)
Comment (gauravyadav-wd) [reply booking-page/index.js]: okay sir

File: booking-page/index.js (hunk 3 above)
Comment (gauravyadav-wd) [reply booking-page/index.js:40]: sir, here I have to loop just based on index and not on an array. Is for loop not fine for that and i variable is just doing index work there so what name should be there instead ?

File: booking-page/index.html (hunk 2 above)
Comment (gauravyadav-wd) [reply booking-page/index.html:118]: Sir, these are seat quantities actually. I am making them and prices dynamic

**Pull request merged**